docs/pages/technology/openid4vci.rst

```diff
@@ -26,11 +26,20 @@ We aim to support other flows and features in future:
 - Client authentication, depending on evolving security requirements.
 
 Enabling
-########
+********
+
+.. note::
+
+    These steps to enable OpenID4VCI are subject to change.
 
 By default, the feature is disabled.
 
-To enable issuing and receiving credentials over OpenID4VCI, set ``vcr.oidc4vci.enabled`` to ``true``.
+To enable issuing and receiving credentials over OpenID4VCI:
+
+- set ``vcr.oidc4vci.enabled`` to ``true``
+- set ``vcr.oidc4vci.url`` to the base URL of your node-to-node API, e.g. ``https://nutsnode.example.com/`` (excluding ``/n2n``).
+  This will typically be base URL of the ``auth.publicurl`` configuration.
+
 To receive credentials over OpenID4VCI for a DID, you also have to register your wallet metadata URL on its DID document.
 You do so by registering a service of type ``oidc4vci-wallet-metadata`` with the ``serviceEndpoint`` pointing to the wallet metadata URL,
 e.g.: ``https://example.com/identity/<did>/.well-known/openid-credential-wallet``
```

Comment:
It is not clear to me that this is the url to the /n2n path. Adding the missing /n2n to the path in the example on the last line already helps